Sources/Passes/Models/PassModel.swift

```diff
@@ -79,4 +90,13 @@ internal extension PassModel {
         
         return authenticationToken
     }
+
+    var _$userPersonalization: OptionalParent<UserPersonalizationType> {
+        guard let mirror = Mirror(reflecting: self).descendant("_userPersonalization"),
+            let userPersonalization = mirror as? OptionalParent<UserPersonalizationType> else {
+                fatalError("userPersonalization property must be declared using @OptionalParent")
```

Comment:
It might be worth using a throwing computed property here? (These were not available when Fluent 4 was released)

Reply:
If I make them throwing, when I use them inside `query.filter()`s it throws this error: `Cannot form key path to property with 'throws' or 'async'`

Reply:
Ah ok, stick with the current way then